### ISSUE TITLE
Resolve differences between top level composition doc and package information section

### ISSUE THREAD

**Issue by nishakm** [spdx/spdx-spec#668]
For v2.3, there are discrepancies between the [spdx composition document](https://github.com/spdx/spdx-spec/blob/development/v2.3/chapters/composition-of-an-SPDX-document.md#522-package-information-section-) and the [package information document](https://github.com/spdx/spdx-spec/blob/development/v2.3/chapters/package-information.md).

**Comment by nishakm** [comment]
@kestewart I'll submit a separate PR for this. It depends on #622 being merged.
